manifests/params.pp

```diff
@@ -11,7 +11,7 @@
   $repo_proxy_password         = 'absent'
   $repo_proxy_username         = 'absent'
   $repo_release                = undef
-  $repo_url_suffix             = '8.x'
+  $repo_url_suffix             = '10.x'
```

Comment:
is this change required? If so it qualifies as a backwards-incompatble change?

Reply:
https://github.com/voxpupuli/puppet-nodejs/pull/414#issuecomment-575895848 :)

Reply:
yeah, I'm fine with 12

Reply:
There we go... :green_heart: